frontend/app/services/user-service.server.ts

```diff
@@ -0,0 +1,52 @@
+import moize from 'moize';
+import { z } from 'zod';
+
+import { getAuditService } from '~/services/audit-service.server';
+import { getInstrumentationService } from '~/services/instrumentation-service.server';
+import { getEnv } from '~/utils/env.server';
+import { getLogger } from '~/utils/logging.server';
+
+const log = getLogger('user-service.server');
+
+export const getUserService = moize(createUserService, { onCacheAdd: () => log.info('Creating new user service') });
+
+function createUserService() {
+  const { CDCP_API_BASE_URI } = getEnv();
+
+  async function getUser(userId: string) {
+    const auditService = getAuditService();
+    const instrumentationService = getInstrumentationService();
+    auditService.audit('alert-subscription.get', { userId });
+    const url = new URL(`${CDCP_API_BASE_URI}/api/v1/users/${userId}`);
```

Comment:
There's no need to create a new `user-service` in my opinion. 

We are already calling GET `/api/v1/users/${userId}` in `subscription-service` -> `getSubscription(..)` so why not just extract the data we want (ie. `email` and `emailVerified`) from that response and return it there instead?